data/partners/partners.yaml

```diff
@@ -3,89 +3,93 @@
   logo: /logos/pkg/azure-native.svg
 
-- name: Oracle
-  link: https://www.oracle.com/cloud/
-  logo: /logos/tech/oracle.svg
+- name: Google Cloud
+  link: https://cloud.google.com/find-a-partner/partner/pulumi
+  logo: /logos/pkg/gcp.svg
 
-- name: Docker
-  link: https://www.docker.com/
-  logo: /logos/pkg/docker.svg
+- name: 1Password
+  link: https://developer.1password.com/docs/pulumi/
+  logo: /logos/tech/1password.svg
 
-- name: Pinecone
-  link: https://docs.pinecone.io/integrations/pulumi
-  logo: /logos/pkg/pinecone.svg
+- name: Checkly
+  link: https://www.checklyhq.com/docs/integrations/pulumi/
+  logo: /logos/tech/checkly.svg
 
-- name: NewRelic
-  link: https://newrelic.com/instant-observability/pulumi/
-  logo: /logos/tech/newrelic.svg
+- name: Chronosphere
+  link: https://docs.chronosphere.io/administer/infrastructure/pulumi
+  logo: /logos/tech/chronosphere.svg
 
-- name: Snyk
-  link: https://docs.snyk.io/integrate-with-snyk/vulnerability-management-tools/pulumi-integration/
-  logo: /logos/tech/snyk.png
+- name: Civo
+  link: https://www.civo.com/
+  logo: /logos/pkg/civo2.svg
 
 - name: Cloudflare
   link: https://developers.cloudflare.com/pulumi/
   logo: /logos/pkg/cloudflare.svg
 
-- name: GitHub
-  link: https://github.com/
-  logo: /logos/pkg/github.svg
-
-- name: Snowflake
-  link: https://www.snowflake.com/
-  logo: /logos/pkg/snowflake.svg
-
-- name: Zscaler
-  link: https://www.zscaler.com/
-  logo: /logos/tech/zscaler.svg
+- name: Dagger
+  link: https://dagger.io/
+  logo: /logos/tech/dagger.svg
 
 - name: Datadog
-  link: https://www.datadoghq.com/
+  link: https://docs.datadoghq.com/integrations/pulumi/
   logo: /logos/tech/datadog.svg
 
+- name: Docker
+  link: https://www.docker.com/blog/pulumi-and-docker-build-cloud/
+  logo: /logos/pkg/docker.svg
+
+- name: GitHub
+  link: https://github.com/marketplace/actions/pulumi-cli-action
+  logo: /logos/pkg/github.svg
+
 - name: GitLab
-  link: https://about.gitlab.com/
-  logo: /logos/pkg/gitlab.svg
-  
-- name: Nitric 
+  link: https://about.gitlab.com/blog/2024/01/10/managing-gitlab-resources-with-pulumi/
+  logo: /logos/tech/gitlab-text.png
+
+- name: NewRelic
+  link: https://newrelic.com/instant-observability/pulumi/
+  logo: /logos/tech/newrelic.svg
+
+- name: Nitric
   link: https://nitric.io/blog/why-nitric-chose-pulumi
   logo: /images/pulumiup-2023/speaker-logos/nitric.svg
 
-- name: Tailscale 
-  link: https://tailscale.com/
-  logo: /logos/pkg/tailscale.svg
+- name: Oracle
+  link: https://docs.oracle.com/en/learn/oci-pulumi-iac/index.html
+  logo: /logos/tech/oracle.svg
 
-- name: Chronosphere 
-  link: https://chronosphere.io/
-  logo: /logos/tech/chronosphere.svg
+- name: Pinecone
+  link: https://docs.pinecone.io/integrations/pulumi
+  logo: /logos/pkg/pinecone.svg
 
-- name: 1Password
-  link: https://1password.com/
-  logo: /logos/tech/1password.svg
+- name: Snowflake
+  link: https://www.snowflake.com/
```

Comment:
Investigate why we were removed from their partner page.

Reply:
I don’t currently have a Snowflake contact but would be happy to engage if you can connect me with the right person. Let me know!